lib/firmenwissen/session.rb

```diff
@@ -0,0 +1,26 @@
+module Firmenwissen
+  module Session
+    module_function
+
+    def cookies
+      @cookies ||= {}
+    end
+
+    def update_from_set_cookie_headers(set_cookie_headers)
+      return if set_cookie_headers.nil?
+
+      set_cookie_headers.each do |header|
+        name_and_value, _other_attrs = header.split(/;\s?/)
+        name, value = name_and_value.split('=', 2)
+
+        next if name.empty? || value.nil? || value.empty?
+
+        cookies[name] = value
+      end
+    end
+
+    def to_cookie
```

Comment:
💅 I'd prefer the name `to_cookie_string`/`to_cookie_str`, but that's optional

Reply:
Interestingly enough I changed it back and forth between `to_cookie` and `to_cookie_string` in the past days. I went along with `to_cookie` for brevity, but I am happy to _take the precision road_.